src/main/cc/wfa/virtual_people/core/model/utils/distributed_consistent_hashing_test.cc

```diff
@@ -99,48 +100,43 @@ TEST(DistributedConsistentHashingTest, TestNormalize) {
   // 1         0.4                           0.2
   // 2         0.4                           0.2
   // 3         0.4                           0.2
-  auto distribution = absl::make_unique<std::vector<DistributionChoice>>();
-  distribution->emplace_back(0, 0.8);
-  distribution->emplace_back(1, 0.4);
-  distribution->emplace_back(2, 0.4);
-  distribution->emplace_back(3, 0.4);
+  std::vector<DistributionChoice> distribution({
+    DistributionChoice({0, 0.8}),
+    DistributionChoice({1, 0.4}),
+    DistributionChoice({2, 0.4}),
+    DistributionChoice({3, 0.4})
+  });
   ASSERT_OK_AND_ASSIGN(
       std::unique_ptr<DistributedConsistentHashing> hashing,
       DistributedConsistentHashing::Build(std::move(distribution)));
-  absl::flat_hash_map<int32_t, int> output_counts = {
-      {0, 0},
-      {1, 0},
-      {2, 0},
-      {3, 0}};
+  absl::flat_hash_map<int32_t, double> output_counts;
   for (int seed = 0; seed < kSeedNumber; ++seed) {
     int32_t output = hashing->Hash(std::to_string(seed));
-    // The output should always be one of [0, 1, 2, 3].
-    EXPECT_TRUE(output_counts.find(output) != output_counts.end());
+    if (output_counts.find(output) == output_counts.end()) {
```

Comment:
I think this whole if-statement is unnecessary. When you do `output_counts[some_key]`, it will insert a new default value at that key if the key does not yet exist. For doubles, this is 0.

Reply:
Done.